nautilus_trader/common/actor.pyx

```diff
@@ -26,19 +26,22 @@ attempts to operate without a managing `Trader` instance.
 
 import asyncio
 from concurrent.futures import Executor
+from typing import Callable
 
 import cython
 
+from cpython.datetime cimport datetime
+from libc.stdint cimport uint64_t
```

Comment:
As above, previous location was correct (Cython will go soon, so thankfully we don't need to remember this for much longer).